source/type/type.h

```diff
@@ -548,7 +548,7 @@ struct CommercialMode : public Header, Nameable{
 
 struct PhysicalMode : public Header, Nameable{
     const static Type_e type = Type_e::PhysicalMode;
-    double co2_emission = 0.;
+    double co2_emission = invalid_idx;
```

Comment:
invalid_idx is a uint_32_t, co2 emission is a double so the remark of vincent is valid, you have to use is directly